src/js/pages/services/DeploymentsTab.js

```diff
@@ -1,11 +1,73 @@
+import moment from 'moment';
 import React from 'react';
+import {Table} from 'reactjs-components';
 
 import AlertPanel from '../../components/AlertPanel';
+import EventTypes from '../../constants/EventTypes';
+import MarathonStore from '../../stores/MarathonStore';
+import ResourceTableUtil from '../../utils/ResourceTableUtil';
+import StringUtil from '../../utils/StringUtil'
+
+const columnClassName = ResourceTableUtil.getClassName;
```

Comment:
Please rename the const, as the name kinda suggests that its simply holding class name (`string`) but it's actually a reference to the class name getter. 
BTW: Do we really need to cache this? I don't see any real performance improvements in doing so.